third-party/cmake/DownloadAndExtractFile.cmake

```diff
@@ -22,7 +22,7 @@ set(SRC_DIR ${PREFIX}/src/${TARGET})
 
 # Check whether the source has been downloaded. If true, skip it.
 # Useful for external downloads like homebrew.
-if(EXISTS "${SRC_DIR}" AND IS_DIRECTORY "${SRC_DIR}")
+if(ALLOW_EXISTING_SRC_DIR AND EXISTS "${SRC_DIR}" AND IS_DIRECTORY "${SRC_DIR}")
```

Comment:
Dear Florian Walch,

The email sent to danielm@tinyhub.tk with subject
"Re: [neovim] [RFC] Travis: Rebuild third-party deps if updated in"
will be delivered as soon as possible.

Your email address is automatically added to the
recipients 'whitelist'. This means for further
emails to danielm@tinyhub.tk, you will not receive a delivery
notification again.

TK MAILIAS
*************************************************
Looking for a way to Protect Your Identity and 
for a great no-more-spam solution for your email
address reply@reply.github.com? Sign up for TK Mailias.

It's free! Please visit http://www.mail.tk/